src/lib/reducer.ts

```diff
@@ -0,0 +1,186 @@
+import type { ComponentsObject, HttpMethods, OASDocument, TagObject } from 'rmoas.types';
+
+import jsonPath from 'jsonpath';
+import jsonPointer from 'jsonpointer';
+import { version as getAPIDefinitionVersion } from 'oas-normalize/dist/lib/utils';
+
+export type ReducerOptions = {
+  /** An array of tags in the OpenAPI definition to reduce by. */
+  paths?: Record<string, string[] | '*'>;
+  /** A key-value object of path + method combinations to reduce by. */
+  tags?: string[];
```

Comment:
I think these descriptions should be switched:
```suggestion
  /** A key-value object of path + method combinations to reduce by. */
  paths?: Record<string, string[] | '*'>;
  /** An array of tags in the OpenAPI definition to reduce by. */
  tags?: string[];
```

Reply:
:ope: